_data/simple-icons.json

```diff
@@ -3654,12 +3655,12 @@
         {
             "title": "Google Maps",
             "hex": "4285F4",
-            "source": "https://upload.wikimedia.org/wikipedia/commons/a/a9/Google_Maps_icon.svg"
+            "source": "https://about.google/brand-resource-center/logos-list/"
```

Comment:
I'm not sure this is a good source replacement since the logo is in PNG in this page.

Reply:
changed to source from commons wiki